addons/mrp_subcontracting/tests/test_subcontracting.py

```diff
@@ -852,7 +852,6 @@ def test_multiple_component_records_for_incomplete_move(self):
         self.assertRecordValues(move._get_subcontract_production(), [
             {'product_qty': 5, 'state': 'done'},
             {'product_qty': 2, 'state': 'done'},
-            {'product_qty': 3, 'state': 'cancel'},
```

Comment:
Could be nice to ensure there is still a production order cancelled for 3 quantity